llvm/lib/Transforms/Vectorize/VPlanVerifier.cpp

```diff
@@ -160,10 +160,6 @@ bool VPlanVerifier::verifyVPBasicBlock(const VPBasicBlock *VPBB) {
     errs() << "Same IR basic block used by multiple wrapper blocks!\n";
     return false;
   }
-  if (IRBB != IRBB->getPlan()->getPreheader()) {
```

Comment:
Extend the check for all expected usages of VPIRBB's?

Reply:
Done, thanks!